### PR TITLE
briefcase cuff-to-wrist functionality + storage locking + storage key

### LINKED ISSUE #31283: FR: Cuff briefcase to wrist
#### what
Apply handcuff to briefcase, briefcase is now glued to hand. Should be relatively simple to code, literally just move the cuffs into a var on the case and set the case to `can_drop = 0` or whatever

#### why
Some IAA came up to me yesterday and asked me for cuffs, to see if this was possible. I guess it's a feature on othercodebases? It's a good one, either way. Also, since orderlies carry briefcases, they'd probably benefit from it as well.

### PR DESCRIPTION
**Pull request by SonixApache**
resolves #31283
couldn't think of any other sanity / fringe cases to check
also added a var that lets you lock storage objects, so they cannot be opened or emptied
and then added a universal but object-adjustable key that lets you toggle the lock because why not

~~This only applies to normal briefcases and not *secure* briefcases because they're not subtypes of briefcases for some reason~~
Also, and this is the case with buddycuffing as well; you can't normal handcuff people that have a briefcase cuffed to their hand.
 - [x] todo: figure out how/if to accommodate uncuffing
 - [x] todo: move code up to parent to accommodate secure briefcases

:cl:
 * rscadd: You can now cuff briefcases to your wrist - drag them onto your sprite while in your hand.

### CODE REVIEW

**Comment by SonixApache** [comment]
oh wait i left in all the to_chats

File: code/game/objects/items/weapons/storage/storage.dm [hunk 1]
```diff
@@ -32,6 +32,7 @@
 	var/internal_store = 0
 	var/list/no_storage_slot = new/list()//if the item is equipped in a slot that is contained in this list, the item will act purely as a clothing item and not a storage item (ie plastic bags over head)
 	var/rustle_sound = "rustle"
+	var/storage_locked = FALSE //you can't interact with the contents of locked storage
```
Comment (Intigracy) [inline code/game/objects/items/weapons/storage/storage.dm:35]: Examining should show whether it's locked or not

**Comment by calzilla1** [comment]
What about the Lawyer's SMG briefcase?

**Comment by SonixApache** [comment]
the syndie SMG briefcase is a briefcase and as such works

**Comment by Dacendeth** [comment]
If I chop it off someones arm and pick it up, is it going to be glued to me?

**Comment by SonixApache** [comment]
That's one of the edge cases I thought of; the cuff stuff disengages on drop.

**Comment by SonixApache** [comment]
Tried moving up the code, it looked horrible and required too much fuckery to get the secure briefcases to work, so copypaste it is. Also added the examine thing.

**Comment by Kurfursten** [comment]
Update wiki, perhaps on Unsecret Tips or the IAA page or somewhere that discusses cuffs

**Comment by Eneocho** [comment]
@Kurfursten, added to the unsecret tips.

**Pull request merged**